admin/js/common.js

```diff
@@ -562,13 +565,13 @@ var qTranslateX=function(pg)
 								}
 							}
 						}else{
-							continue;
+							continue;
```

Comment:
git should convert line breaks correctly, if you checkout the whole file with git and then put it back with git too? Interesting, it is not too important, but there must be a way to deal with it ...

Were mixed line breaks there to begin with? I may need to check if there are mixed linebreaks in all the files. I thought PHP would issue warning on this, would not it?